rpa-builder/6.0/modules/ROOT/pages/using-workflow-debugger.adoc

```diff
@@ -0,0 +1,57 @@
+= Using Workflow Debugger
+
+RPA Builder provides an integrated debugger for Workflows. To use it, click the *Debug Workflow* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) or press _F8_. This starts a test run and opens Workflow Debugger instead of stopping the execution of the Workflow if an error occurs.
+
+When you execute a *Debug Workflow* run, you can call Workflow Debugger by pressing _Pause/Break_ on your keyboard. You can change the configuration of Workflow objects within the Workflow Debugger and then re-run them or continue from the current position. When the Workflow enters an error state, Workflow Debugger shows.
```

Comment:
"When the Workflow enters an error state, Workflow Debugger shows." Should there be something at the end of this sentence?